client/src/components/Manifest/Transporter/TransporterRowActions.tsx

```diff
@@ -54,7 +54,7 @@ export function TransporterRowActions({
         swapTransporter(index, index - 1);
       },
       disabled: isFirst,
-      label: `move transporter ${index} up`,
+      label: `move transporter ${index + 1} up`,
```

Comment:
Great find!

Reply:
Thanks! Wasn’t sure if it was appropriate to include in this PR, but figured it was a small cosmetic change in the same directory and went with it.